src/earn_app/_response.py

```
@@ -0,0 +1,830 @@
+        if (
+            inspect.isclass(
+                origin  # pyright: ignore[reportUnknownArgumentType]
+            )
+            and not issubclass(origin, BaseModel)
+            and issubclass(origin, pydantic.BaseModel)
+        ):
+            raise TypeError("Pydantic models must subclass our base model type, e.g. `from earn_app import BaseModel`")
+
+        if (
+            cast_to is not object
+            and not origin is list
```

Flake8 (7.2.0)

[error] test for object identity should be 'is not'

(E714)